### PR TITLE
Feature column resizing

### PR DESCRIPTION
**Pull request by mbrn**
## Related Issue

Relate the Github issue with this PR using `#`

## Description

Simple words to describe the overall goals of the pull request's commits.

## Related PRs

List related PRs against other branches:

| branch              | PR       |
| ------------------- | -------- |
| other_pr_production | [link]() |
| other_pr_master     | [link]() |

## Impacted Areas in Application

List general components of the application that this PR will affect:

\*

## Additional Notes

This is optional, feel free to follow your hearth and write here :)

### CODE REVIEW

File: demo/demo.js [hunk 1]
```diff
@@ -522,56 +519,55 @@ class App extends Component {
                       };
                     },
                   }}
-                  editable={{
```
Comment (Domino987) [inline demo/demo.js:525]: You changed quite a lot of the demo. Was that on purpose?

**Review by Domino987 — COMMENTED** [review]
You changed quite a lot of the demo. Was that intentional?
The code looks good, but it does not work as I expect it to work on my machine.

**Comment by dalenhughes** [comment]
this release broke my app using remote data. paging forward then paging backward caused the app to hang/crash with no useful logs in the console. rolling back to 1.68 resolves the issue.

**Comment by wilomgfx** [comment]
Yeah it's a bit odd, why isn't the `columnResizable` options not in the documentation?

**Pull request merged**